migrations/20220321220618-rename-virtual-card-epire-date-to-expiry-date.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+import { omit } from 'lodash';
+
+import { crypto } from '../server/lib/encryption';
+
+module.exports = {
+  async up(queryInterface) {
+    const [virtualCards] = await queryInterface.sequelize.query(
+      `SELECT "id", "privateData" FROM "VirtualCards" WHERE "privateData" IS NOT NULL;`,
+    );
+    console.info(`Renaming 'expireDate' to 'expiryDate' for ${virtualCards.length} VirtualCards...`);
+    try {
+      await queryInterface.sequelize.transaction(async transaction => {
+        for (const virtualCard of virtualCards) {
+          const originalPrivateData = JSON.parse(crypto.decrypt(virtualCard.privateData));
+          if (!('expireDate' in originalPrivateData)) {
```

Comment:
Would have been interesting to pre-filter that with SQL

```sql
SELECT "id", "privateData"
FROM "VirtualCards"
WHERE "privateData" IS NOT NULL
AND "privateData" ? 'expireDate';
```

Reply:
I thought about it too but the data is encrypted.